extensions/src/libadalang-helpers.ads

```diff
@@ -302,6 +309,13 @@ package Libadalang.Helpers is
            (Parser, Long => "--symbolic-traceback",
             Help         => "Show symbolic tracebacks for exceptions");
 
+         package Exit_On_Missing_File is new Parse_Flag
+           (Parser,
+            Short => "-E", Long => "--exit-on-missing-file",
+            Help  => "Behavior when encountering missing files. By default,"
+            & " continue, but if this is passed, will continue and present"
+            & " results anyway.");
```

Comment:
IIUC this has become wrong (“by default continue, otherwise continue” looks suspicious)

Reply:
It was wrong before, and is still wrong eheh. Let me fix it